app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -239,13 +238,7 @@ class HomeFragment : Fragment() {
                     topSites = components.core.topSitesStorage.cachedTopSites,
                     tip = components.strictMode.resetAfter(StrictMode.allowThreadDiskReads()) {
                         FenixTipManager(
```

Comment:
Just making a note, but it would be nice to investigate if we need `FenixTipManager` after this removal.